app/__tests__/popup.test.tsx

```diff
@@ -0,0 +1,106 @@
+///<reference path='./../../node_modules/@types/enzyme/index.d.ts'/>
+///<reference path='./../../node_modules/@types/jasmine/index.d.ts'/>
+///<reference path='./../../node_modules/@types/enzyme-adapter-react-15/index.d.ts'/>
+///<reference path='./../../node_modules/@types/jest/index.d.ts'/>
+import * as React from 'react';
+import { configure } from 'enzyme';
+import { shallow } from 'enzyme';
+import * as Adapter from 'enzyme-adapter-react-15';
+import {Popup} from './../popup';
+
+import * as MessageService from '../message_service';
+
+configure({ adapter: new Adapter() });
```

Comment:
move this to `tests/setup.js` and configure it in jest config `"setupTestFrameworkScriptFile": "<rootDir>/tests/setup.js"`